cadc-test-vos/src/main/java/org/opencadc/conformance/vos/FilesTest.java

```diff
@@ -271,6 +279,100 @@ public void emptyFileTest() throws Exception {
 
     }
 
+    @Test
+    public void testDataNodePermission() {
+
+        Assume.assumeTrue("testDataNodePermission not configured, skipping", altSubject != null);
+
+        try {
+            // Create a DataNode.
+            String nodeName = "files-data-node-permissions";
+            String nodePath = nodeName;
+            URL nodeURL = getNodeURL(nodesServiceURL, nodePath);
+            VOSURI nodeURI = getVOSURI(nodePath);
+            DataNode testNode = new DataNode(nodeName);
+            testNode.owner = authSubject;
+            testNode.isPublic = false;
+            log.debug("files-data-node-permissions: " + nodeURL);
+
+            // cleanup
+            delete(nodeURL, false);
+
+            // PUT the node
+            put(nodeURL, nodeURI, testNode);
+
+            // Create a pullFromVoSpace Transfer
+            Transfer transfer = new Transfer(nodeURI.getURI(), Direction.pullFromVoSpace);
+            transfer.version = VOS.VOSPACE_21;
+            Protocol protocol = new Protocol(VOS.PROTOCOL_HTTPS_GET);
```

Comment:
minor point: you should always add an anon Protocol for systems that use embedded token (like vault)

main point: this is the FilesTest and you are creating a DataNode and doing transfer negotiation; that belongs in TransferTest

This test: HEAD (OK) and GET (DENIED) on the /files endpoint to test GetAction